app/uk/gov/hmrc/merchandiseinbaggage/service/EmailService.scala

```diff
@@ -139,38 +149,60 @@ class EmailService @Inject()(emailConnector: EmailConnector, declarationReposito
       "eori"                      -> eori.value
     )
 
-    val calculationParams = {
-      maybeTotalCalculationResult match {
-        case Some(total) =>
-          Map(
-            "customsDuty" -> total.totalDutyDue.formattedInPounds,
-            "vat"         -> total.totalVatDue.formattedInPounds,
-            "total"       -> total.totalTaxDue.formattedInPounds
-          )
-
-        case None => Map.empty
-      }
-    }
-
     val allParams =
       if (declarationType == DeclarationType.Import)
-        goodsParams ++ commonParams ++ calculationParams
+        goodsParams ++ commonParams ++ paymentParams(declaration)
       else
         goodsParams ++ commonParams
 
     DeclarationEmailInfo(
       Seq(emailTo),
-      templateId(lang, declarationType),
+      templateId(lang, declarationType, amendments),
       allParams
     )
   }
 
-  private def templateId(lang: String, declarationType: DeclarationType): String = {
-    val importTemplate = if (lang == "en") "mods_import_declaration" else "mods_import_declaration_cy"
-    val exportTemplate = if (lang == "en") "mods_export_declaration" else "mods_export_declaration_cy"
-    if (declarationType == DeclarationType.Import)
-      importTemplate
-    else
-      exportTemplate
+  private def templateId(lang: String, declarationType: DeclarationType, amendments: Seq[Amendment]): String = {
+    val isAmendJourney = amendments.nonEmpty
+
+    val templateId = (declarationType, isAmendJourney) match {
+      case (Import, false) => "mods_import_declaration"
+      case (Import, true)  => "mods_amend_import_declaration"
+      case (Export, false) => "mods_export_declaration"
+      case (Export, true)  => "mods_amend_export_declaration"
+    }
+
+    if (lang == "en") {
+      templateId
+    } else {
+      templateId + "_cy"
+    }
+  }
+
+  case class PaymentMade(totalDutyDue: AmountInPence, totalVatDue: AmountInPence, totalTaxDue: AmountInPence)
+
+  private def paymentParams(declaration: Declaration) = {
+
+    def paymentForCalculation(maybeTotalCalculationResult: Option[TotalCalculationResult]) =
+      maybeTotalCalculationResult match {
+        case Some(total) => PaymentMade(total.totalDutyDue, total.totalVatDue, total.totalTaxDue)
+        case None        => PaymentMade(AmountInPence(0), AmountInPence(0), AmountInPence(0))
+      }
+
+    val declarationPayment = paymentForCalculation(declaration.maybeTotalCalculationResult)
+
+    val amendmentPayments = declaration.amendments.map(a => paymentForCalculation(a.maybeTotalCalculationResult))
```

Comment:
Do you need to filter only amendments that have been paid?

Reply:
good point Andy, i did this check for displaying goods, but looks like i missed it here.. will double check and add it now